Customizing list icons: give a way to apply to current item only (#2454)
MarkBind/markbind: docs/userGuide/syntax/lists.md

If you customize any item on a certain level, you must also customize the first item on that level. If not, the list will revert to its uncustomized form.
</box>

> **You can add an icon that is only for a specific item by using the `i-one-off` attribute.**

Reviewer: Suggest using `one-off` because after the merging of #2444, this attribute will apply to not only the icon attributes :)

Reviewer 2: `i-once` (to make it shorter)?

Author: Would combining the 2 and calling it `once` fine?

Reviewer: I think once is ok! @damithc I think not having the `i-` is better because this should apply to the `t-` attributes as well

If you customize any item on a certain level, you must also customize the first item on that level. If not, the list will revert to its uncustomized form.
</box>

**You can add an icon that is only for a specific item by using the `i-one-off` attribute.**

<include src="codeAndOutput.md" boilerplate >
<variable name="highlightStyle">markdown</variable>
<variable name="code">
* Item 1 { icon="glyphicon-education" }

> * Item 2 { icon="fas-code-branch" i-one-off="true" }

Reviewer: ```suggestion
* Item 2 { icon="fas-code-branch" i-one-off=true }
```
Let's encourage users to use a boolean if possible - the string is just to make it more robust

  * Item 2.1
  * Item 2.2
* Item 3
  </variable>
</include>

> Setting `i-one-off=true` is preferred but `i-one-off="true"` works as well.

Reviewer: Move this line below and put it in a tip box

Reviewer: tbh I think we don't even need this. We shouldn't be encouraging this behaviour in the first place to use a string over boolean.

You may need to add escape characters when using special characters for Markdown in text. 
</box>

> **You can add an icon or text that is only for a specific item by using the `once` attribute.**

Reviewer: Can you also add the fact that other attributes also won't get inherited? And add an example for this?

<include src="codeAndOutput.md" boilerplate >
<variable name="highlightStyle">markdown</variable>
<variable name="code">
* Item 1 { icon="glyphicon-education" }
* Item 2 { icon="fas-code-branch" once=true }
  * Item 2.1
  * Item 2.2
* Item 3
  </variable>
</include>

> Subsequent items at the same level will not inherit the once icon or text and will instead inherit the icon or text before it.

Reviewer: ```suggestion
Subsequent items at the same level will not inherit the `once` attributes and will instead inherit the attribute before it.
```

You may need to add escape characters when using special characters for Markdown in text. 
</box>

**You can add an icon or text that only applies to a specific item by using the `once` attribute.**

<include src="codeAndOutput.md" boilerplate >
<variable name="highlightStyle">markdown</variable>
<variable name="code">
* Item 1 { icon="glyphicon-education" }
* Item 2 { icon="fas-code-branch" once=true }
  * Item 2.1
  * Item 2.2
* Item 3
  </variable>
</include>

Subsequent items at the same level will not inherit icons or texts with the `once` attribute and will instead inherit the icons or texts before it.

The `once` attribute also applies to all other attributes such as `i-size` and `t-size` as seen below.

<include src="codeAndOutput.md" boilerplate >
<variable name="highlightStyle">markdown</variable>
<variable name="code">
* Item 1 { icon="glyphicon-education" i-size="40px" i-class="text-primary" }

> * Item 2 { i-size="25px" once=true}

Reviewer: ```suggestion
* Item 2 { i-size="25px" once=true }
```
